tests/unit_tests/test_wave_06.py

```diff
@@ -143,7 +156,16 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory 
+    assert item_c not in tai.inventory 
```

Comment:
Same as before! Try and stay consistent if you are going to include the `not in` assertion!